src/kern/exec.c

```diff
@@ -53,6 +68,32 @@ int BPF_PROG(test_void_hook, struct linux_binprm *bprm)
 	// Increment the previous value, as a new process was executed on the
 	// current CPU.
 	(*val)++;
+
+	// Get information about the current process
+	int pid = bpf_get_current_pid_tgid() >> 32;
```

Comment:
Also keep the declarations separate so 

int pid, tgid, ppid; 

and then separately initialise them.